parachain/test-parachains/adder/collator/Cargo.toml

```diff
@@ -36,4 +45,6 @@ substrate-test-utils = { git = "https://github.com/paritytech/substrate", branch
 sc-service = { git = "https://github.com/paritytech/substrate", branch = "master" }
 sp-keyring = { git = "https://github.com/paritytech/substrate", branch = "master" }
 
+polkadot-node-core-pvf = { path = "../../../../node/core/pvf" }
+
```

Comment:
```suggestion
```

Reply:
we still don't need it both as a dev-dep and a regular one?

Reply:
ah right, we only need to have it as a regular dep.